business-central-tests/business-central-tests-gui/src/main/java/org/kie/wb/selenium/model/KieWbDistribution.java

```diff
@@ -22,8 +22,7 @@
 public enum KieWbDistribution {
 
     KIE_WB,
```

Comment:
Please rename this enum constant `KIE_WB -> BUSINESS_CENTRAL` and the enum class `KieWbDistribution -> BusinessCentralDistribution`